packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeDefinition.ts

```diff
@@ -569,6 +562,19 @@ export class ModelsTreeDefinition implements HierarchyDefinition {
   }
 }
 
+/**
+ * Creates a list of `HierarchyNodeIdentifierPath` that can be used to filter the hierarchy
+ * to only display nodes that either match the provided instance keys and their children or contain a provided label.
+ * @beta
+ */
+export async function createInstanceKeyPaths(props: ModelsTreeInstanceKeyPathsProps) {
```

Comment:
- This changes a number of APIs from internal to public - please ensure all of them have appropriate release tags.
- Now that this function is moved away from `ModelsTreeDefinition`, it's name should contain `ModelsTree`.

Reply:
With the addition of passing `createInstanceKeyPaths` to `getFilteredPaths`, this is no longer necessary.

Reply:
The new way exposes `ElementsGroupInfo` and now needs a release tag.

Reply:
Added a release tag.